src/Xenial.Tasty/FeatureDetection/Win/ProcessExtensions.Toolhelp32.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+
+namespace Xenial.Delicious.FeatureDetection.Win
+{
+    internal static class Toolhelp32
```

Comment:
[CA1060](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1060?view=vs-2019)?

Reply:
Hm its internal and I like the idea of feature detection. So i guess guideline can be bended.

How so we enable CA warnings?

Reply:
https://docs.microsoft.com/en-us/visualstudio/code-quality/install-fxcop-analyzers?view=vs-2019